### DISCUSSION TITLE
This product is amazing

### DISCUSSION THREAD

**Discussion by reynadan** [community/community#43289] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hello,

Just wanted to thank github for this marvel.
I can do whatever I want with my codespace on any device and it's really easy to use with the extensions.
I'll drop my cloud desk and I cannot wait to see the future of this.

Happy new year!

**Comment by erikaellr** [comment]
Hi @reynadan, we're thrilled that you're using Codespaces and having such a good experience with it! We are always working to make GitHub and Codespaces as awesome as possible, so please continue to feel free to share whatever feedback you have!
